core/assets/jsons/global.json

```diff
@@ -72,14 +90,27 @@
     "tint": [1.0, 1.0, 1.0, 1.0],
     "startframe":       0
   },
-  "wall": {
+  "vertical-wall": {
```

Comment:
Why the name switch between "vertical-wall" and "single-wall"? Why not just one or the other?

Reply:
Good point, will fix it